airbyte-integrations/connectors/source-s3/source_s3/source_files_abstract/stream.py

```diff
@@ -488,8 +373,6 @@ def get_updated_state(self, current_stream_state: MutableMapping[str, Any], late
         )
         state_dict[self.cursor_field] = datetime.strftime(max(current_parsed_datetime, latest_record_datetime), self.datetime_format_string)
 
-        state_dict["schema"] = self._get_schema_map()
```

Comment:
no need to store schema - inference is fast now and merging schemas is no more supported